src/mmw/apps/modeling/views.py

```diff
@@ -216,18 +218,17 @@ def _initiate_tr55_job_chain(model_input, job_id):
 def _construct_tr55_job_chain(model_input, job_id):
     job_chain = []
 
-    current_hash = model_input['modification_hash']
-    census = model_input.get('census')
-    census_hash = None
+    if True:  # XXX
```

Comment:
I would prefer commenting out the block that you want to keep rather than putting it into an unreachable `else` block.